cloud/deployment/inspect/inspect.go

```diff
@@ -0,0 +1,157 @@
+	DeploymentURL  string    `mapstructure:"deployment-url"`
+	WebserverURL   string    `mapstructure:"webserver-url"`
+	UpdatedAt      time.Time `mapstructure:"updated-at"`
+	Status         string    `mapstructure:"status"`
+}
+
+type deploymentConfig struct {
+	Name              string              `mapstructure:"name"`
+	Description       string              `mapstructure:"description"`
+	ClusterID         string              `mapstructure:"cluster-id"` // this is also in deploymentInfo
+	RunTimeVersion    string              `mapstructure:"runtime-version"`
+	SchedulerAU       int                 `mapstructure:"scheduler-au"`
+	SchedulerReplicas int                 `mapstructure:"scheduler-replicas"`
+	AlertEmails       []string            `mapstructure:"alert-emails"`
+	WorkerQueues      []astro.WorkerQueue `mapstructure:"worker-queues"`
+	AstroVariables    Variables           `mapstructure:"astronomer-variables"`
+}
+
+type PrintableDeployment struct {
+	DeploymentInfo   map[string]interface{} `mapstructure:"deployment-info"`
+	DeploymentConfig map[string]interface{} `mapstructure:"deployment-config"`
+}
+
+type Variables struct {
+	EnvFromFile  string
+	VariableList []astro.EnvironmentVariablesObject `mapstructure:"astronomer-variables"`
+}
+
+var (
+	jsonMarshal = json.MarshalIndent
+	yamlMarshal = yaml.Marshal
+)
+
+const (
+	jsonFormat = "json"
+)
+
+func Inspect(wsID, deploymentName, deploymentID, outputFormat string, client astro.Client, out io.Writer) error {
+	var (
+		requestedDeployment                    astro.Deployment
+		err                                    error
+		infoToPrint                            []byte
+		deploymentInfoMap, deploymentConfigMap map[string]interface{}
+	)
+	// get or select the deployment
+	requestedDeployment, err = deployment.GetDeployment(wsID, deploymentID, deploymentName, client)
+	if err != nil {
+		return err
+	}
+
+	deploymentInfoMap, err = getDeploymentInspectInfo(&requestedDeployment)
+	if err != nil {
+		return err
+	}
+
+	deploymentConfigMap, err = getDeploymentConfig(&requestedDeployment)
+	if err != nil {
+		return err
+	}
+
+	infoToPrint, err = formatPrintableDeployment(deploymentInfoMap, deploymentConfigMap, outputFormat)
+	if err != nil {
+		return err
+	}
+	fmt.Fprint(out, string(infoToPrint)+"\n")
+	return nil
+}
+
+func getDeploymentInspectInfo(sourceDeployment *astro.Deployment) (map[string]interface{}, error) {
+	var (
+		deploymentURL string
+		c             config.Context
+		err           error
+	)
+	c, err = config.GetCurrentContext()
+	if err != nil {
+		return nil, err
+	}
+
+	deploymentURL = "cloud." + c.Domain + "/" + sourceDeployment.Workspace.ID + "/deployments/" + sourceDeployment.ID + "/analytics"
+
+	return map[string]interface{}{
+		"deployment-id":   sourceDeployment.ID,
+		"workspace-id":    sourceDeployment.Workspace.ID,
+		"cluster-id":      sourceDeployment.Cluster.ID,
+		"airflow-version": sourceDeployment.RuntimeRelease.AirflowVersion,
```

Comment:
does it work wit AC deployments? I mean As a cloud user can i use this inspect command?

Reply:
cc @neel-astro

Reply:
I don't think so cloud platform supports AC images anyways, that's just a software thing I believe

Reply:
Just so I can learn what is AC deployment?